code/modules/mob/new_player/new_player.dm

```diff
@@ -275,6 +275,9 @@
 		return 0
 	if(!job.player_old_enough(src.client))
 		return 0
+	if(job.species_whitelist.len)
```

Comment:
I took that from the preferences menu job choosing thing, I don't actually know if it's needed